gtfu/src/main/java/gtfu/Recipients.java

```diff
@@ -0,0 +1,33 @@
+package gtfu;
+import java.io.*;
+import java.util.*;
+
+public class Recipients {
+
+	private static final String RECIPIENTS_FILE_PATH = "src/main/resources/conf/recipients.txt";
+	private static final String DELIMITER = ":";
+	private Map<String, String> recipientsMap;
+
+	public Recipients() {
+
+        recipientsMap = Util.hashMapFromTextFile(RECIPIENTS_FILE_PATH,DELIMITER);
+	}
+
+    public String[] get(String reportType) {
+
+        String recipientsList = null;
+        // Default to local recipient list if env variable is present
+        recipientsList = System.getenv("LOCAL_REPORT_RECIPIENTS");
+
+        if (recipientsList == null) {
+        	recipientsList = recipientsMap.get(reportType);
+
+			if (recipientsList == null) {
+				System.out.println("** error: no report type " + reportType + " listed in " + RECIPIENTS_FILE_PATH);
+				System.exit(1);
```

Comment:
Calling `System.exit()` is a very bad idea for code that isn't a top-level application. You don't know who might be calling this in the future, and there is no stack trace to explain what happened. Just throw an exception.